feature/home/src/main/java/com/epmedu/animeal/home/presentation/viewmodel/HomeViewModel.kt

```diff
@@ -65,13 +68,6 @@ class HomeViewModel @Inject constructor(
         }
     }
 
-    /*private fun fetchGpsSettingsUpdates() {
```

Comment:
Remove commented call to this method above

Reply:
It is already removed